linux-agents-userdata.tpl

```diff
@@ -44,7 +44,7 @@ Description=Jenkins Agent
 [Service]
 User=jenkins
 WorkingDirectory=/var/lib/jenkins
-ExecStart=/usr/bin/java -jar /swarm-client.jar -master https://${dns_name}.${dns_base_name} -tunnel ${dns_name}-master.${dns_base_name}:43863 -fsroot /var/lib/jenkins -executors 1 -username agents -passwordEnvVariable JENKINS_AGENTS_PASSWORD
+ExecStart=/usr/bin/java -jar /swarm-client.jar -master https://internal.${dns_name}.${dns_base_name}:8080 -tunnel internal.${dns_name}.${dns_base_name}:43863 -fsroot /var/lib/jenkins -executors 1 -username agents -passwordEnvVariable JENKINS_AGENTS_PASSWORD
```

Comment:
Should this be `http://internal`? Not sure that it matters

Reply:
In my use case, I needed to allow public access to the Jenkins ALB. Though instead of using the ALB to establish the connection I'm setting up a new internal URL just to allow the 'join' without using the ALB.

Reply:
I double-checked your comment. You're already creating the DNS with a different name. No need to create an internal nor anything else.